README.md

```diff
@@ -69,7 +86,19 @@ AZNFS is supported on following Linux distros:
 	```
 	sudo mount -t aznfs -o vers=3 <account-name>.blob.core.windows.net:/<account-name>/<container-name> /mountpoint
 	```
-- Logs generated from AZNFS can be found in `/opt/microsoft/aznfs/data/aznfs.log`.
+- Mount the Azure File NFS share using following command:
+	```
+	sudo mount -t aznfs -o vers=4.1 <account-name>.file.core.windows.net:/<account-name>/<container-name> /mountpoint
+	```
+   For air-gapped environments, ensure that the environment varialbe "AGEndPoint" is set to the appropriate endpoint before running the mount command:
```

Comment:
Remove all mention of air-gapped and AG. It is internal to Microsoft. We need to use something more generic. For the environment variable, something like "AzureEndpointOverride" might work.  Others might have better ideas.

Also, fix the "varialbe" spelling.